### PR TITLE
[EDGENT-327] Use local Javadoc URLs

### PR DESCRIPTION
**Pull request by queeniema**

### CODE REVIEW

File: site/_data/mydoc/mydoc_topnav.yml [hunk 1]
```diff
@@ -60,24 +60,24 @@ javadoc_dropdowns:
       version: all
       output: web
       items:
-        - title: lastest
-          external_url: http://edgent.incubator.apache.org/javadoc/latest/index.html
+        - title: latest
+          url: /..\/javadoc\/latest
```
Comment (dlaboss) [inline site/_data/mydoc/mydoc_topnav.yml:64]: newbie question: why the backslash characters?

File: site/_config.yml [hunk 2]
```diff
@@ -126,9 +126,9 @@ host:    127.0.0.1
...
 downloadsurl: /docs/downloads
 projurl: /
-#docsurl: /javadoc/latest
+docsurl: /javadoc/latest
 # above yields things like the following when running on a "jekyll serve" test server
 # [2016-12-22 15:23:31] ERROR `/favicon.ico' not found.
 # [2016-12-22 15:33:17] ERROR `/javadoc/latest/resources/fonts/dejavu.css' not found.
 # the following is undesirable as it doesn't link to the "jekyll serve" test server
-docsurl: https://edgent.incubator.apache.org/javadoc/latest
+# docsurl: https://edgent.incubator.apache.org/javadoc/latest
```
Comment (dlaboss) [inline site/_config.yml:134]: Do the changes in this PR eliminate the error msgs when selecting the latest javadoc?  Just remove all of the comments about the problem as well as the undesired URL?

File: site/_data/mydoc/mydoc_topnav.yml (hunk 1 above)
Comment (queeniema) [reply site/_data/mydoc/mydoc_topnav.yml:64]: When testing locally, Jekyll wasn't recognizing the path without the backslashes. Does `url: /javadoc/latest` work for anyone else?

File: site/_config.yml (hunk 2 above)
Comment (queeniema) [reply site/_config.yml:134]: I still see the error messages related to `dejavu.css`, but the Javadoc looks correct. Should we not commit these changes until we can get around the messages?

Comment (dlaboss) [reply site/_config.yml:134]: Yeah, I thought the javadoc looked correct too but wasn't confident they were completely benign.  But even if benign, I think it's a bad idea to commit something that's generating them.  At best folks will be later be surprised / confused by them.

Comment (queeniema) [reply site/_config.yml:134]: Makes sense, I'll close this pull request and try to get rid of those messages. Thanks!

**Pull request closed without merging**